.github/workflows/ci.yml

```diff
@@ -82,6 +82,15 @@ jobs:
         with:
           version: v2.4.0
 
+  vulnerability-scan:
+    name: Vulnerability Scan
+    uses: ./.github/workflows/vulncheck.yml
+    permissions:
+      security-events: write
+    with:
+      target-branch: ${{ github.event.pull_request.base.ref || github.ref_name }}
+      go-version-input: '1.24.10'
```

Comment:
Is there a way to get the golang version from the go.mod file like how we do it for the setup-go action on line 72?